packages/docs/docs/guides/9-request storage subgraph/0-entities-storage.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Storage Subgraph Entities
+keywords: [Subgraph, Entities, Schema]
+description: Subgraph entities and descriptions.
+---
+
+# Transaction
+
+Description: get specific details of the transaction
+
+| Field                | Type      | Description                                                                                  |
+| -------------------- | --------- | -------------------------------------------------------------------------------------------- |
+| id                   | ID!       | hash of the first transaction in the channel                                                 |
```

Comment:
Please revert this to "a unique ID based on IPFS hash & position of the transaction in the document"